test/factories/groups.rb

```diff
@@ -20,6 +20,19 @@
 
   factory :group_with_parent, parent: :group do
     parent { create(:group) }
+
+    after(:create) do |group|
+      sibling_group = create(:group)
+      sibling_group.update(parent: group.parent)
+    end
+
+    after(:create) do |group|
+      user = create(:user)
+      Membership::CreateService.call(group, user)
+      membership = user.membership_for(group)
+      CreatePost.call(group, membership, PostType::EVALUATION_HELPER_ID)
+      ActivityNotification::Notification.last.delete
```

Comment:
` Membership::CreateService.call` sends a notification to the group leader. 
We sholud delete the sent notification beacause it can mess up test where we check howmany notifications are sent by performing an aciton.